### ISSUE TITLE
Support cpp wrapper code

### ISSUE THREAD

**Issue by jgong5** [pytorch/torchdynamo#1556]
Currently wrapper.py generates python code that invokes generated kernels and external kernels. This would incur Python overhead which can be avoided if the wrapper can generate c++ code that invokes these kernels via C++ directly.

**Comment by voznesenskym** [comment]
> Currently wrapper.py generates python code that invokes generated kernels and external kernels. This would incur Python overhead which can be avoided if the wrapper can generate c++ code that invokes these kernels via C++ directly.

Yes. It could also run where we don't have access to Python. In fact, this would go great with torchdynamo.export, for a general no Python inductor flow. 

Do you plan on working on direct c++ invocation? If not, I can take a look.

**Comment by jgong5** [comment]
> Do you plan on working on direct c++ invocation? If not, I can take a look.

It is in our tentative plan. It's great if you'd like to offer the hand. Feel free to let me know if you need any help!

**Comment by jgong5** [comment]
@voznesenskym Have you started working on it? If not, we will take over. :-)
